### PR TITLE
LPS-129924	[FE] Show paragraph field on Sidebar

### PR DESCRIPTION
**Pull request by miltonmc**
- Ticket: [LPS-129924](https://issues.liferay.com/browse/LPS-129924
)
Co-authored-by: Aline D'Oleron <aline.cantarelli@liferay.com>
Co-authored-by: Elivelton Rodrigues <elivelton.rodrigues@liferay.com>

### CODE REVIEW

**Comment by miltonmc** [comment]
ci:reevaluate:1047401_7873

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/plugins/field-sidebar/components/FormsFieldSidebar.es.js [hunk 1]
```diff
@@ -105,7 +105,9 @@ export const FormsFieldSidebar = ({title}) => {
 			displaySettings={Object.keys(focusedField).length > 0}
 			editingLanguageId={editingLanguageId}
 			fieldTypes={sortFieldTypes(
-				fieldTypes.filter(({group}) => group === 'basic')
+				fieldTypes.filter(
+					({group}) => group === 'basic' || group === 'interface'
+				)
```
Comment (matuzalemsteles) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/plugins/field-sidebar/components/FormsFieldSidebar.es.js:110]: From what I understand in the old Sidebar, he put it together in `Basic Elements` belonging to the group `basic` and `interface` and added any other group in `Customized Elements` which is the custom fields that customers can add, to support this, the best thing here would be to remove the filter and just pass `fieldTypes`.

cc @natocesarrego what do you think?

**Comment by miltonmc** [comment]
ci:test:sf

**Comment by miltonmc** [comment]
ci:test:sf

**Comment by miltonmc** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:forward

**Pull request closed without merging**